compiler/damlc/tests/daml-test-files/SetFieldParens.EXPECTED.renamed-daml

```diff
@@ -1,3 +1,4 @@
-SetFieldParens.foo : (Int, Bool) -> (Int, Bool)
-SetFieldParens.foo tup
-= (setField @"_2" not (getField @"_2" tup) tup)
+SetFieldParens.foo : (Text -> (Int, Bool)) -> (Int, Bool)
+SetFieldParens.foo mkTup
+= (setField
+@"_2" (not (getField @"_2" (mkTup "baz"))) (mkTup "bar"))
```

Comment:
whatever is formatting this (could be the pretty printer GHC uses for `-ddump-rn`, or Shake's wrapper around warnings, or something else) seems to drop any indentation. At least it seems to be consistent between OSs, but probably not great if we want to add larger tests for renamer output.